public/Scripts/Game.js

```diff
@@ -1428,11 +1490,11 @@ var ticksPerSecond=20,game,GameController=function()
 		this.generation=n;
 		this.id=t;
 		this.gender=i;
-		this.traits.push(new Trait(0,"vitality",5));
-		this.traits.push(new Trait(1,"strength",5));
-		this.traits.push(new Trait(2,"agility",5));
-		this.traits.push(new Trait(3,"bite",5));
-		this.traits.push(new Trait(4,"sting",5))
+		this.traits.push(new Trait(0,"vitality",50));
```

Comment:
Shouldn't this be reverted?  My guess is you don't want to change the balance that much :P